glide.yaml

```diff
@@ -0,0 +1,18 @@
+package: .
+homepage: https://github.com/HewlettPackard/oneview-golang
+import:
+- package: github.com/docker/machine
+  version: v0.8.0-rc2
+  subpackages:
+  - libmachine/log
+- package: github.com/stretchr/testify
+  version: v1.1.3
+  subpackages:
+  - assert
+excludeDirs:
```

Comment:
I was using `excludeDirs` because i was getting errors like the following when fetching dependencies:

```
[INFO]	Resolving imports
[INFO]	Fetching github.com/HewlettPackard/oneview-golang/rest into /home/docker/git/github.com/HewlettPackard/oneview-golang/vendor
[ERROR]	Error looking for github.com/HewlettPackard/oneview-golang/rest: Unable to get repository
[INFO]	Found Godeps.json file in vendor/github.com/docker/machine
[INFO]	--> Parsing Godeps metadata...
[ERROR]	Error scanning github.com/HewlettPackard/oneview-golang/utils: open /home/docker/git/github.com/HewlettPackard/oneview-golang/vendor/github.com/HewlettPackard/oneview-golang/utils: no such file or directory
[ERROR]	Error scanning github.com/HewlettPackard/oneview-golang/liboneview: open /home/docker/git/github.com/HewlettPackard/oneview-golang/vendor/github.com/HewlettPackard/oneview-golang/liboneview: no such file or directory
[ERROR]	Error scanning github.com/HewlettPackard/oneview-golang/testconfig: open /home/docker/git/github.com/HewlettPackard/oneview-golang/vendor/github.com/HewlettPackard/oneview-golang/testconfig: no such file or directory
```

Is there a better way to handle this when doing a glide install or update?